core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -237,6 +240,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeOptionalWriteable(fuzziness);
         out.writeVInt(prefixLength);
         out.writeVInt(maxExpansions);
+        out.writeBoolean(fuzzyTranspositions);
```

Comment:
This break wire backwards compatibility which we can't do. Have a look at https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/action/bulk/byscroll/BulkByScrollTask.java#L219-L221 for an example of how we keep it.